docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes pod creation requests. For each container that does not specify compute resource requests or limits,
```

Comment:
I think this sentence is ambiguous, maybe write: "For each container that specifies neither compute resource requests nor limits,"